ocaml/xapi-aux/kerberos_encryption_types.ml

```diff
@@ -28,6 +28,21 @@ module Winbind = struct
     | All ->
         "all"
 
+  let to_int = function
+    (*
+     * [X] 0x00000001 DES-CBC-CRC
+     * [X] 0x00000002 DES-CBC-MD5
+     * [X] 0x00000004 RC4-HMAC
+     * [X] 0x00000008 AES128-CTS-HMAC-SHA1-96
+     * [X] 0x00000010 AES256-CTS-HMAC-SHA1-96
+     * *)
+    | Strong ->
+        24
+    | Legacy ->
```

Comment:
I don't see why you'd specifically want *only* legacy, and not any of the newer types, that'll just make upgrading the encryption type difficult. 
i.e. why do we need Legacy, and not just All and Strong?
Is there a situation where All wouldn't work, and only Legacy would?

Reply:
To keep align with *kerberos encryption types (G)*
https://www.samba.org/samba/docs/4.9/man-html/smb.conf.5.html

Reply:
But Legacy here also activates DES, which seems unintended?

Reply:
NO, this is intended to be compatible with old legacy DC.

Reply:
We should match exactly what Samba does then and make Legacy=rc4_hmac (i.e. 0x04 only).
If someone really needs DES, they can set it to All instead.